app/models/commonTaskList/SelfEmploymentTitles.scala

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.commonTaskList
+
+import models.common.{BusinessId, TaxYear, WithName}
+import play.api.libs.json.{Json, OWrites, Reads}
+
+object SelfEmploymentTitles {
+
+  val values: List[TaskTitle] = List(
+    TradeDetails(), // TODO should trade details be excluded, or maybe in it's own section?
+    SelfEmploymentAbroad(),
```

Comment:
This ticket has lots of TODOs because SE isn't properly mapped up to the common task list yet. So will need to come back and make changes once we know what changes we need to make.